src/apps/loan-list/helpers.ts

```diff
@@ -16,6 +16,14 @@ export const getAmountOfRenewableLoans = (list: LoanV2[]) => {
   return list.filter(({ isRenewable }) => isRenewable).length;
 };
 
+export const sortByLoanDate = (list: LoanV2[]) => {
+  return list.sort(
+    (objA, objB) =>
+      new Date(objA.loanDetails.loanDate).getTime() -
+      new Date(objB.loanDetails.loanDate).getTime()
+  );
+};
+
 export const formatDate = (date: string) => {
```

Comment:
Think I have mentioned it in another PR. But I think this function belongs to the global date helper file in core.

Reply:
I'll move it and merge